src/main/java/com/fasterxml/jackson/dataformat/xml/util/Stax2Util.java

```diff
@@ -0,0 +1,31 @@
+package com.fasterxml.jackson.dataformat.xml.util;
+
+import org.codehaus.stax2.XMLInputFactory2;
+import org.codehaus.stax2.io.Stax2ByteArraySource;
+import org.codehaus.stax2.io.Stax2CharArraySource;
+
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import java.io.ByteArrayInputStream;
+import java.io.CharArrayReader;
+
+public class Stax2Util {
```

Comment:
Javadoc with `@since`?

Reply:
Is it ok to just say that the small stax2 API jar is a mandatory dependency? Then no refactor is needed.

Reply:
Yes: I would consider it a non-optional dependency. Does not bring any Stax(2) implementation, should not (AFAIK) cause additional problems.

Although I am bit curious about that comment that alluded to some problems.